### PR TITLE
chore(unlock): notice on unlock page to recommend chrome

### PR DESCRIPTION
**Pull request by WanderingHogan**
**What this PR does** 📖
Adds a warning message on unlock page to recommend Chrome, but still allow user to log in/create account

Screen Shot 2022-08-25 at 1 41 45 PM

**Which issue(s) this PR fixes** 🔨

**Special notes for reviewers** 🗒️
this is hidden on mobile, and removes the realms since we dont have realms rightnow. Firefox on desktop does not trigger the mobile flag when resized like chromium browsers do

**Additional comments** 🎤

### CODE REVIEW

File: locales/en-US.js [hunk 1]
```diff
@@ -238,6 +238,8 @@ export default {
       store_pin: 'Store Pin? (Less Secure)',
       delete_account_label: 'Not you? Create or import an account',
       generate_random_user: 'Generate a Random User (DEV)',
+      browser_warning:
+        'Hey there, welcome to our early access! While one of our goals is to make Satellite work across all browsers, we have focused the early access on Chromium based browsers. You are welcome to try Satellite on any browser you like, and we will focus on more broad support soon, but you may run in to more UI and functionality issues.',
```
Comment (stavares843) [inline locales/en-US.js:242]: > Hey there, welcome to our early access! While one of our goals is to make Satellite work across all browsers, we have focused the early access on Chromium-based browsers. You are welcome to try Satellite on any browser you like, and we will focus on more broad support soon, but you may run into more UI and functionality issues.


suggestion

**Comment by stavares843** [comment]
Captura de ecrã 2022-08-25, às 20 54 19


using safari, this gets cut off, maybe we can divide in multiple lines instead

File: locales/en-US.js (hunk 1 above)
Comment (WanderingHogan) [reply locales/en-US.js:242]: cool, can you make that change?

**Comment by stavares843** [comment]
also, i only get the warning on safari

doesn't appear on opera or edge

**Comment by WanderingHogan** [comment]
good find on safari

opera, brave, edge are all chromium based so we dont want the warning there

**Comment by stavares843** [comment]
thanks for clarifying 🔨

File: locales/en-US.js (hunk 1 above)
Comment (stavares843) [reply locales/en-US.js:242]: will update 🔨

Comment (stavares843) [reply locales/en-US.js]: done 

https://github.com/Satellite-im/Core-PWA/pull/4475/commits/558b121c880559a169a956412f67b5df3523d9df

**Review by stavares843 — APPROVED** [review]

**Pull request merged**